ha/virsh/tests/fence_virsh_test.sh

```diff
@@ -14,7 +14,7 @@ oneTimeSetUp() {
 }
 
 test_cluster_nodes_are_online() {
-  cluster_status=$(run_command_in_node "${IP_VM01}" "sudo crm status")
+  cluster_status=$(run_command_in_node "${IP_VM01}" "sudo pcs cluster status")
```

Comment:
In most places you use `pcs status` instead of `pcs cluster status`. I don't mind either way, but it's probably best to be consistent.

Reply:
The output of `pcs status` and `pcs cluster status` are different as far as I remember. Here I am using `pcs cluster status` because I am checking if the nodes are online. With `pcs status` you get resources information, but yes, with `pcs status` I think you also have information about nodes.